R/schema_CellData.R

```diff
@@ -1,3 +1,22 @@
+# Why not `new("CellData", ...)`? It seems excessive to store the schema as
+# an attribute for each cell. Possibly a premature concern.
+new_CellData <- function(...) {
+  # explicit 'list' class is a bit icky but it makes jsonlite happy
+  structure(rlang::list2(...), class = c(
+    "googlesheets4_schema_CellData", "googlesheets4_schema", "list"
+  ))
+}
+
+# Use this instead of `new_CellData()` when (light) validation makes sense.
+CellData <- function(...) {
+  dots <- rlang::list2(...)
```

Comment:
FWIW we normally do/recommend `@import rlang`

Reply:
OK will do that separately